au/code/au/apply_magnitude_test.cc

```diff
@@ -40,23 +40,34 @@ TEST(CategorizeMagnitude, FindsIntegerMultiplyInstances) {
     EXPECT_EQ(categorize_magnitude(mag<2>()), ApplyAs::INTEGER_MULTIPLY);
     EXPECT_EQ(categorize_magnitude(mag<35>()), ApplyAs::INTEGER_MULTIPLY);
 
+    EXPECT_EQ(categorize_magnitude(-mag<35>()), ApplyAs::INTEGER_MULTIPLY);
```

Comment:
Usual comment about `EXPEC_THAT`. 😁

Reply:
I don't want to add a lot of noise in this PR, so here's my counter-offer:

- I filed #404 to track migrating the whole repo.
- I'll provide a fast-follow PR, after this one lands, that fully migrates every file where you left a comment like this one in this PR.

Good deal?  :slightly_smiling_face: